docs/buttons/button-group.mdx

```diff
@@ -9,7 +9,7 @@ components: ButtonGroup
 
 <Label>Buttons</Label>
 
-# <span name="floating-nav">ButtonGroup</span>
+# <HeaderDocCustom title="ButtonGroup" url="https://main--62ffec7466615c40c8dbe435.chromatic.com/?path=/story/buttons-buttongroup--button-group" />
```

Comment:
the baseUrl should be a environment variable and we need pass only the path of them (components)

@lautarovilas